packages/bits/src/lib/expander/expander.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="nui-expander"
      [class.nui-expander--expanded]="state === 'expanded'">
     <div class="nui-expander__header"
+         role="switch"
```

Comment:
The closes widget to the expander is Disclosure. For the disclodure type of elements, we have a set of requirements
https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure

See the documentation for the details